app/javascript/sdk/IFrameHelper.js

```diff
@@ -166,6 +175,9 @@ export const IFrameHelper = {
     iframe.style.visibility = '';
     iframe.setAttribute('id', `chatwoot_live_chat_widget`);
 
+    const closeBtn = document.getElementById('closeBtn');
+    closeBtn.style.visibility = '';
```

Comment:
We can maybe set this to `visible` make set this explicitly 🤷 

Tho was I wrote that out I see that convention here might be to use the empty string so maybe better to do that!

Reply:
I think I'll keep the empty string to try to be consistent with their code